src/NzbDrone.Host/AccessControl/SslAdapter.cs

```diff
@@ -12,7 +12,7 @@ public interface ISslAdapter
 
     public class SslAdapter : ISslAdapter
     {
-        private const string APP_ID = "C2172AF4-F9A6-4D91-BAEE-C2E4EE680613";
+        private const string APP_ID = "C2172AF4-F9A6-4D91-BAEE-C2E4EE680615";
```

Comment:
Should this be completely randomly generated?

Reply:
I don't know that it matters, but I can generate another random one in VS. I googled it to make sure nothing obvious was using it lol.